Userland/Libraries/LibWeb/DOMTreeModel.cpp

```diff
@@ -118,8 +120,8 @@ static String with_whitespace_collapsed(const StringView& string)
 GUI::Variant DOMTreeModel::data(const GUI::ModelIndex& index, GUI::ModelRole role) const
 {
     auto const& node = *static_cast<JsonObject const*>(index.internal_data());
-    auto node_name = node.get("name").as_string();
-    auto type = node.get("type").as_string_or("unknown");
+    auto node_name = node.get("name"sv).as_string();
+    auto type = node.get("type"sv).as_string_or("unknown");
```

Comment:
Please separate these changes into a new commit.

Reply:
That seems overkill, peppering `sv` here is the whole reason for this commit? If anything I'd just change the commit title to `LibWeb: Use StringView literals in DOMTreeModel` or something.

Reply:
I was talking about separating all the `sv`s to their own commit, separate from the icon additions.

Reply:
Ohh sorry that was indeed separate, I was looking at the changes tabs instead of each commit individually. My bad.

Reply:
okay, no problem.^^